### DISCUSSION TITLE
Request add new Code snippet

### DISCUSSION THREAD

**Discussion by thosanmaytinh** [hoppscotch/hoppscotch#3252] [General]
Can you add the functionality to create a code snippet for curl from a request?

```
curl --location 'https://domain.com' \
--header 'Content-Type: application/json' \
--header 'Accept-Encoding: application/json' \
--header 'Authorization: Basicxxxx' \
--data '{"query":"....}","variables":{}}'
```

**Comment by liyasthomas** [comment]
This feature is implemented in Hoppscotch.

Import a cURL request to Hoppscotch by simply pasting it on the URL input bar on the REST page. Or alternatively, click on the “More options” drop-down button right next to “Send” > “Import cURL”.
